ExcelReaderAPI/Services/DatabaseHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Data.SqlClient;
+namespace ExcelReaderAPI.Services
+{
+    public class DatabaseHelper
+    {
+        private readonly string _connectionString;
+
+        public DatabaseHelper(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public string GetDBConnString()
+        {
+            var dbConnString = _connectionString;
```

Comment:
no need of variable, you can return directly

Reply:
resolved